credentials-operator/values.yaml

```diff
@@ -112,3 +113,4 @@ global:
 # If that Deployment is owned by a custom resource, the operator will not be able to resolve it.
 # For this resolving to be successful, the operator needs to be able to `get` all resources.
 allowGetAllResources: true
+databaseSecretRotationIntervalNanoseconds: ""
```

Comment:
Viper supports a "duration" string here, which means it can also be "24h". Users are likely to need a format that's not just nanoseconds (closer to hours/days). So let's rename this to `duration` and make sure we document (in otterize/docs) how to use with different units.